src/test/java/me/devyonghee/lotto/model/LottoStoreTest.java

```diff
@@ -0,0 +1,33 @@
+package me.devyonghee.lotto.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+@DisplayName("로또 상점")
+class LottoStoreTest {
+
+    @Test
+    @DisplayName("객체화")
```

Comment:
테스트 설명에는 어떤조건에서 어떤 결과가 유추된다는 내용이 추상적으로 작성되어 있으면, 테스트의 갯수가 늘어날수록 직접 코드 분석이 아닌, 콘솔창의 결과를 보고 유추가 가능해져서 좋습니다. 
가령 예를 들어, 이 테스트는 다음과 같이 설명할 수 있을 것 같아요.

```suggestion
    @DisplayName("유효한 인수를 전달할 경우 로또상점은 정상적으로 생성된다.")
```

Reply:
코멘트 감사합니다. 이부분도 수정하도록 하겠습니다!